recipe_scrapers/keukenliefdenl.py

```diff
@@ -0,0 +1,85 @@
+# mypy: allow-untyped-defs
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class KeukenLiefdeNL(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "keukenliefde.nl"
+
+    def author(self):
+        return self.soup.find("meta", {"name": "author"}).get("content")
+
+    def title(self):
+        return self.soup.find("meta", {"property": "og:title"}).get("content")
+
+    def category(self):
+        return self.soup.find(
+            "div", {"class": "article-meta-item sp gerecht"}
+        ).getText()
+
+    def total_time(self):
+        return get_minutes(
+            self.soup.find("div", {"class": "article-meta-item sp tijd"}).get_text()
+        )
+
+    def yields(self):
+        yields = self.soup.find("div", {"class": "article-meta-item sp aantal"})
+        if yields:
+            return get_yields(yields.get_text())
+
+        return None
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredents_container = self.soup.find("div", {"id": "clipboard-ingredients"})
+        if ingredents_container:
+            return self.process_ingredients(ingredents_container)
+
+        ingredient_header = self.soup.find("strong", string="Ingrediënten")
+        if ingredient_header:
+            return self.process_ingredients(
+                ingredient_header.parent.find_next_sibling("ul")
+            )
+
+        # Nothing found, we give up.
+        return []
```

Comment:
The code and tests are looking good!

A small change: in the case of both `ingredients` and `instructions`, I think we should consider it an error if no items are found.  A recipe by definition should have at least one instruction, and at least one ingredient).  The generic `RecipeScrapersExceptions` exception is the one in both cases, with a suitable error message.

(it looks like we have one other scraper that does this; I'll take a note to adjust that similarly)

Reply:
Added `ElementNotFoundInHtml` which is what we can't do. The elements are not found in html.